### PR TITLE
3.1 - first website

### PR DESCRIPTION
**Pull request by easmith2**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+html {
+  min-width: 690px;
+  min-height: 435px;
+}
```
Comment (rpearce) [inline styles.css]: I don't recommend setting widths & heights like this on the `html` tag. Instead, I would create a `.container` `div` to wrap around what you need

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+.details {
+  border: 2px solid #F3F3F3;
+  border-top: none;
+  border-bottom-right-radius: 8px;
+  border-bottom-left-radius: 8px;
+  text-align: center;
+}
+
+div detail {
```
Comment (rpearce) [inline styles.css]: what is `detail`?

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  width: 280px;
+  height: 240px;
+  margin: 20px 10px 20px 10px;
+}
+.photo {
+  width: 280px;
+  height: 240px;
+  vertical-align: bottom;
+  border-top-right-radius: 8px;
+  border-top-left-radius: 8px;
```
Comment (rpearce) [inline styles.css]: i find the singular `border-radius` tag to be awesome:
```scss
border-radius: 8px 8px 0 0
```

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.title {
+  display: block;
+  text-align: center;
```
Comment (rpearce) [inline styles.css]: if all the text in this is going to be centered, you could just put this on the `.politician_card` class and not repeat yourself. Or, if you wanted to keep it in these classes only, you could do a grouped selector
```scss
.title,
.details {
  text-align: center;
}
```

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+div detail {
+  display: inline-block;
+  vertical-align: middle;
+  margin: 2% 16% 2% 16%;
+}
+
+* {
+  color: #4D4D4D;
+  font-family: Avenir, Seravek, Trebuchet MS, sans-serif;
+  font-weight: lighter;
```
Comment (rpearce) [inline styles.css]: I didn't know this was a thing. I recommend using the `X00` values (like `100`, `200`, `300`, `400` ... etc)

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  font-size: 14px;
+  -webkit-box-sizing: border-box;
+     -moz-box-sizing: border-box;
+          box-sizing: border-box;
```
Comment (rpearce) [inline styles.css]: I don't think you need the vendor prefixes...

http://caniuse.com/#feat=css3-boxsizing

**Comment by rpearce** [comment]
Groovy. :+1:

File: styles.css (hunk 2 above)
Comment (easmith2) [inline styles.css]: voodoo magic? 
I initially tried having each 'detail' in it's own `div`, but was having trouble getting them on the same line with the text centered. I saw something somewhere in the abyss of the interwebs that was formatted this way, so I tried it and it worked. I didn't realize at the time that I was basically making a custom tag, which I'm guessing is bad. (It sounds bad, at least.)
I updated it to be a div with a detail class (not to be confused with the details class, which is no longer used anywhere) and it works just fine.

**Pull request merged**